src/screens/simulation/common.js

```diff
@@ -54,6 +54,33 @@ export const formatParametersForModel = ({ start_date, ...parameters }) =>
         start_time: 0,
     });
 
+export const formatRulesForModel = (rules = [], parameters) =>
```

Comment:
why `rules = []` ?

Reply:
Just to not send an null list to the API